bot/internal/bot/bot.go

```diff
@@ -153,6 +159,42 @@ func classifyChanges(e *env.Environment, files []github.PullRequestFile) env.Cha
 	return ch
 }
 
+// approverCount returns the number of required approvers for the PR by comparing the files included
+// in the PR against a set of paths that only require a single approver and the PR author against
+// a set of authors that only require a single approver. 1 is returned when all of the
+// files match a single approver path or the PR author matches one of the authors, otherwise
+// env.DefaultApproverCount is returned.
+func approverCount(authors, paths []string, author string, files []github.PullRequestFile) int {
+	if len(files) == 0 {
+		return env.DefaultApproverCount
+	}
+
+	// check if pr author only requires a single approval
+	for _, a := range authors {
+		if author == a {
+			return 1
+		}
+	}
```

Comment:
```suggestion
	if slices.Contains(authors, author) {
		return 1
	}
```

Reply:
My muscle memory hasn't gotten used to the slices package yet. I need to code more :/